tests/testthat/test-expect_identical_linter.R

```diff
@@ -30,6 +30,10 @@ test_that("expect_identical_linter skips cases likely testing numeric equality",
   lint_msg <- rex::rex("Use expect_identical(x, y) by default; resort to expect_equal() only when needed")
 
   expect_lint("expect_equal(x, 1.034)", NULL, linter)
+  expect_lint("expect_equal(x, -1.034)", NULL, linter)
+  expect_lint("expect_equal(x, c(-1.034))", NULL, linter)
+  expect_lint("expect_equal(x, -c(1.034))", NULL, linter)
```

Comment:
Is this case relevant in practice?

Reply:
I'm on the fence.

On the one hand, it should be caught be `unnecessary_concatenation_linter()` and converted to the standard case. On the other hand, it's not just a false negative; not treating this case would lead us to produce incorrect advice.